tshirts.c

```diff
@@ -15,6 +15,7 @@ char size(int cms) {
 
 int main() {
     assert(size(37) == 'S');
+    assert(size(38) == 'S'||size(38) == 'M');

```

Comment:
Do you expect size 38 to be S or M?
you need to clearly state the expected behavior (whatever you assume) using asserts and cover all range of values including boundary values.